### PR TITLE
A quick hack so we can visualize the 'tabatkins' grammar

### PR DESCRIPTION
**Pull request by sfkleach**
This Python program uses the library for rendering the `pop11_grammar_tabatkins.txt` file.

### CODE REVIEW

**Review by willprice — APPROVED** [review]
Would be nice to agree on a common templating language in python to use across projects, but apart from that, no comments :) Good job, we can set up a CI job down the line to generate these!

File: tabakins2html.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+def eval_expression(input_string, allowed_names = {}):
+    code = compile(input_string, "<string>", "eval")
+    for name in code.co_names:
+        if name not in allowed_names:
+            raise NameError(f"Use of {name} not allowed")
+    return eval(code, {"__builtins__": {}}, allowed_names)
+
+def pars_tadatkins_file():
```
Comment (willprice) [inline tabakins2html.py]: it's tabatkins (same issue persistent throughout)

Comment (willprice) [reply tabakins2html.py]: plus pars -> parse

File: tabakins2html.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        if current:
+            dump.append( current )
+    return map( lambda L: ''.join(L), dump )
+
+def diagram2string( diag ):
+    with io.StringIO() as buffer:
+        diag.writeSvg(buffer.write)   
+        return buffer.getvalue()
+
+def svgList():
```
Comment (willprice) [inline tabakins2html.py]: svgList -> svg_list to be consistent within file.

File: tabakins2html.py (hunk 1 above)
Comment (sfkleach) [reply tabakins2html.py]: Fixed - thanks - I had a bit of dyslexia there!

File: tabakins2html.py (hunk 2 above)
Comment (sfkleach) [reply tabakins2html.py]: Fixed

**Comment by willprice** [comment]
Just noticed, the filename is misspelled too: `tabakins2html.py` should be `tabatkins2html.py`

**Pull request merged**